### ISSUE TITLE
Security Warnings - OpenVX

### ISSUE THREAD

**Issue by kiritigowda** [GPUOpen-ProfessionalCompute-Libraries/MIVisionX#958]
```
MIVisionX/amd_openvx/openvx/ago/ago_drama_divide.cpp:604:20: warning: 'sprintf' is deprecated: This function is provided for compatibility reasons only.  Due to security concerns inherent in the design of sprintf(3), it is highly recommended that you use snprintf(3) instead. [-Wdeprecated-declarations]
                                char desc[64]; sprintf(desc, "scalar-virtual:UINT8,%d", anode->attr_border_mode.constant_value.U8);
                                              ^
```

**Comment by LakshmiKumar23** [comment]
@kiritigowda do we still see this warning?

**Comment by paveltc** [comment]
@kiritigowda I built MIVIsionX TOT master and develop yesterday and did not see this warning. I'll take a look at the the PR though and comment there.

**Comment by kiritigowda** [comment]
Fixed with #1122
